packages/clerk-js/src/ui/components/OrganizationSwitcher/UserInvitationSuggestionList.tsx

```diff
@@ -0,0 +1,193 @@
+      onClick={handleAccept}
+      localizationKey={localizationKeys('organizationSwitcher.action__suggestionsAccept')}
+    />
+  );
+};
+
+const AcceptRejectInvitationButtons = (props: UserOrganizationInvitationResource) => {
+  const card = useCardState();
+  const { userInvitations } = useCoreOrganizationList({
+    userInvitations: organizationListParams.userInvitations,
+  });
+
+  const handleAccept = () => {
+    return card
+      .runAsync(props.accept)
+      .then(removeItemFromPaginatedCache(userInvitations))
+      .catch(err => handleError(err, [], card.setError));
+  };
+
+  return (
+    <Button
+      elementDescriptor={descriptors.organizationSwitcherInvitationAcceptButton}
+      textVariant='buttonExtraSmallBold'
+      variant='outline'
+      colorScheme='neutral'
+      size='sm'
+      isLoading={card.isLoading}
+      onClick={handleAccept}
+      localizationKey={localizationKeys('organizationSwitcher.action__invitationAccept')}
+    />
+  );
+};
+
+const InvitationPreview = withCardStateProvider(
+  (
+    props: PropsWithChildren<{
+      publicOrganizationData:
+        | UserOrganizationInvitationResource['publicOrganizationData']
+        | OrganizationSuggestionResource['publicOrganizationData'];
+    }>,
+  ) => {
+    const { children, publicOrganizationData } = props;
+    return (
+      <Flex
+        align='center'
+        gap={2}
+        sx={t => ({
+          minHeight: 'unset',
+          height: t.space.$12,
+          justifyContent: 'space-between',
+          padding: `0 ${t.space.$6}`,
+        })}
+      >
+        <OrganizationPreview
+          elementId='organizationSwitcher'
+          avatarSx={t => ({ margin: `0 calc(${t.space.$3}/2)` })}
+          organization={publicOrganizationData}
+          size='sm'
+        />
+
+        {children}
+      </Flex>
+    );
+  },
+);
+
+const SwitcherInvitationActions = (props: PropsOfComponent<typeof Flex> & { showBorder: boolean }) => {
+  const { showBorder, ...restProps } = props;
+  return (
+    <Actions
+      sx={t => ({
+        borderTop: showBorder ? `${t.borders.$normal} ${t.colors.$blackAlpha200}` : 'none',
+      })}
+      {...restProps}
+    />
+  );
+};
+
+export const UserInvitationSuggestionList = () => {
+  const { ref, userSuggestions, userInvitations } = useFetchInvitations();
+  const isLoading = userInvitations?.isLoading || userSuggestions?.isLoading;
+  const hasNextPage = userInvitations?.hasNextPage || userSuggestions?.hasNextPage;
+  const hasAnyData = !!(userInvitations?.count || userSuggestions?.count);
+  return (
+    <SwitcherInvitationActions
+      showBorder={hasAnyData || isLoading}
+      elementDescriptor={descriptors.organizationSwitcherPopoverInvitationActions}
+    >
+      <Box
+        sx={t => ({
+          maxHeight: `calc(4 * ${t.sizes.$12})`,
+          overflowY: 'auto',
+          ...common.unstyledScrollbar(t),
+        })}
+      >
+        {(userInvitations?.count ?? 0) > 0 &&
```

Comment:
Can we have a top-level condition to avoid the repeated use of `.?`?

Reply:
userInvitations will always be defined, but count may be undefined so I will change this to 
```ts
{(userInvitations.count ?? 0) > 0 &&
```

Do you believe that the default returned value for `userInvitations.count` should be 0 instead of undefined ?